plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoConnectorTest.java

```diff
@@ -346,55 +350,75 @@ public void testJson()
                 "CREATE TABLE test_json_array AS SELECT JSON '[\"a\", \"b\", \"c\"]' AS col",
                 "Can't convert json to MongoDB Document.*");
 
-        assertUpdate("DROP TABLE test_json");
+        assertUpdate("DROP TABLE " + tableName);
     }
 
     @Test
     public void testArrays()
     {
-        assertUpdate("CREATE TABLE tmp_array1 AS SELECT ARRAY[1, 2, NULL] AS col", 1);
-        assertQuery("SELECT col[2] FROM tmp_array1", "SELECT 2");
-        assertQuery("SELECT col[3] FROM tmp_array1", "SELECT NULL");
-
-        assertUpdate("CREATE TABLE tmp_array2 AS SELECT ARRAY[1.0E0, 2.5E0, 3.5E0] AS col", 1);
-        assertQuery("SELECT col[2] FROM tmp_array2", "SELECT 2.5");
-
-        assertUpdate("CREATE TABLE tmp_array3 AS SELECT ARRAY['puppies', 'kittens', NULL] AS col", 1);
-        assertQuery("SELECT col[2] FROM tmp_array3", "SELECT 'kittens'");
-        assertQuery("SELECT col[3] FROM tmp_array3", "SELECT NULL");
-
-        assertUpdate("CREATE TABLE tmp_array4 AS SELECT ARRAY[TRUE, NULL] AS col", 1);
-        assertQuery("SELECT col[1] FROM tmp_array4", "SELECT TRUE");
-        assertQuery("SELECT col[2] FROM tmp_array4", "SELECT NULL");
-
-        assertUpdate("CREATE TABLE tmp_array5 AS SELECT ARRAY[ARRAY[1, 2], NULL, ARRAY[3, 4]] AS col", 1);
-        assertQuery("SELECT col[1][2] FROM tmp_array5", "SELECT 2");
-
-        assertUpdate("CREATE TABLE tmp_array6 AS SELECT ARRAY[ARRAY['\"hi\"'], NULL, ARRAY['puppies']] AS col", 1);
-        assertQuery("SELECT col[1][1] FROM tmp_array6", "SELECT '\"hi\"'");
-        assertQuery("SELECT col[3][1] FROM tmp_array6", "SELECT 'puppies'");
+        String tmpArray1 = "tmp_array1" + randomNameSuffix();
```

Comment:
Leaving suggestions for new variable and table names since the diff isn't clean either. 
```suggestion
        String arrayIntegerTable = "test_array_integer" + randomNameSuffix();
```

We use `test_` prefix for table names in tests.